docs/src/content/docs/request-state.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Request State
+description: Request State
+packageName: '@inox-tools/request-state'
+challenger: [Florian, Oliver]
```

Comment:
What is this thing 😂?

Reply:
Spoiler for a future thing. I'll add a badge on every page indicating who made the challenge